packages/ai-jsx/src/index.ts

```diff
@@ -15,8 +15,9 @@ export interface Element<P> {
   [attachedContext]?: RenderContext;
 }
 
-interface IndirectNode {
-  _magic: never;
+const indirectNodeSymbol = Symbol('AI.indirectNode');
+export interface IndirectNode {
```

Comment:
Can we get block comments on this (and, ultimately, everything else exported from this file)?

Reply:
Added some to (most of) the new stuff